x-pack/plugins/security/public/views/login/login.html

```diff
@@ -44,4 +44,14 @@
     </form>
   </div>
 
+  <div class="euiText loginErrorEsUnavailable" ng-if="login.layout === 'error-es-unavailable'">
+    <p class="euiTitle euiTitle--medium euiTextColor euiTextColor--danger">Cannot connect to the Elasticsearch cluster currently configured for Kibana.</p>
+    <p>Refer to the Kibana logs for more details.</p>
+  </div>
+
+  <div class="euiText loginErrorXpackUnavailable" ng-if="login.layout === 'error-xpack-unavailable'">
+    <p class="euiTitle euiTitle--medium euiTextColor euiTextColor--danger">Cannot connect to an Elasticsearch cluster running the OSS distribution from an instance of Kibana that has a Basic license or above.</p>
```

Comment:
small nit, feel free to ignore: technically we are connected to the cluster and we may eventually be able to speak better. maybe instead along the lines of `kibana has x-pack and elasticsearch doesn't.  please use matching distributions`

Reply:
also to distinguish from the other error message saying elasticsearch is offline  (vs online but can't feature match)

Reply:
@debadair, thoughts here?